buildroot-external/rootfs-overlay/etc/systemd/timesyncd.conf

```diff
@@ -1,3 +1,2 @@
 [Time]
-NTP=time.cloudflare.com
```

Comment:
Why not using the default NTP pools from ntp.org? They would be less prone to be a single point of failure in case Cloudflare goes down since the servers behind ntp.org are from multiple organizations.

Reply:
I don't agree with the fact that this project would be considered a vendor because of a company shipping it preinstalled. it would be either way even without a commercial product. I am keen on considering asking the ntp.org organization to create the "vendor" custom domains `0.home-assistant.pool.ntp.org 1.home-assistant.pool.ntp.org 2.home-assistant.pool.ntp.org 3.home-assistant.pool.ntp.org`.

While I like to be an idealistic person I understand that in this case one should be a little bit more pragmatic. I'll split the problem of what address put in the fallback configuration and place the matter of the default pool in its own issue so we can merge at least the bugfix and continue discussing somewhere else. What do you think on this?